fedbiomed/researcher/secagg/_secure_aggregation.py

```diff
@@ -150,9 +150,7 @@ def __init__(
                 f"but got {type(active)} "
             )
 
-        if clipping_range is not None and (
-            not isinstance(clipping_range, int) or isinstance(clipping_range, bool)
-        ):
+        if clipping_range is not None and not isinstance(clipping_range, int):
             raise FedbiomedSecureAggregationError(
```

Comment:
Thank you @ybouilla 
FYI this is not part of this PR. Please don't check the diffs with `develop` but with `feature/1198-lom-secagg-nonce` in this PR. Thx !

Reply:
Why would you want specifically a test for this case ?
Don't we just want to test it has the proper type (None or int) ?

Reply:
because `isinstance(True, int) ` returns `True`

Reply:
Sorry I don't see the point ...
In python a bool is an int. We probably don't want to test that anytime we check `isinstance(myvar, int)` that `not isinstance(myvar, bool)` ?

Reply:
I thought it could have been useful. But as you prefer, for me it is not straightforward that `clipping_range=Ture` will translate into `clipping_range=1`.
I can understand testing both may be inconvenient, but Do you see any bad reason we should not test both `isinstance(myvar, int)` and `isinstance(myvar, bool)`?

Reply:
IMO if we test it here we should (coherently) do the same test anytime we test `isinstance(something, int)` ... a few dozen times. I agree I don't really like this also, but testing this is really un-pythonic ... unless we have a very specific case where accepting a bool as an int would cause an error.